src/components/swap/confirmPriceImpactWithoutFee.ts

```diff
@@ -12,7 +12,7 @@ export default function confirmPriceImpactWithoutFee(priceImpactWithoutFee: Perc
     return (
       window.prompt(
         i18next.t('swap.priceImpactMinPrompt', { priceImpact: PRICE_IMPACT_WITHOUT_FEE_CONFIRM_MIN.toFixed(0) })
-      ) === 'confirm'
+      ) === i18next.t('swap.confirm')
```

Comment:
let's stick with the convention of importing i18n as `t` and using that reference